arctic/chunkstore/chunkstore.py

```diff
@@ -48,8 +46,9 @@ def _ensure_index(self):
                                       unique=True, background=True)
 
     @mongo_retry
-    def __init__(self, arctic_lib, chunker=DateChunker()):
+    def __init__(self, arctic_lib, chunker=DateChunker(), serializer=NumpyString()):
```

Comment:
The name `NumpyString` seems fishy.  There are a number of other classes inside the serialization package, are any of the others compatible?